notebooks/tutorial/7 - Getting data from the ADX.ipynb

```diff
@@ -0,0 +1,705 @@
+    "    and i_understand_the_estimates_above_may_be_wrong\n",
+    "    and i_am_responsible_for_those_costs\n",
+    "    and i_have_subscribed_to_the_dataset_on_the_adx\n",
+    "    ):\n",
+    "    agreed_to_terms = True\n",
+    "    print(\"Thank you for agreeing to the terms\")\n",
+    "else:\n",
+    "    agreed_to_terms = False\n",
+    "    raise Exception(\"You have not accepted the terms.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37fd1eef",
+   "metadata": {},
+   "source": [
+    "## Import the usual"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc149e14",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pureskillgg_makenew_pyskill.notebook import setup_notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a0ba17f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "setup_notebook()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4011bab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "import warnings\n",
+    "import json\n",
+    "from pureskillgg_dsdk import (\n",
+    "    enable_auto_exporting_adx_dataset_revisions_to_s3,\n",
+    "    disable_auto_exporting_adx_dataset_revisions_to_s3,\n",
+    "    download_adx_dataset_revision,\n",
+    "    get_adx_dataset_revisions,\n",
+    "    export_single_adx_dataset_revision_to_s3,\n",
+    "    export_multiple_adx_dataset_revisions_to_s3,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "411d12df",
+   "metadata": {},
+   "source": [
+    "# 4. Decide what to Transfer\n",
+    "\n",
+    "You must set **transfer_option** in the cell below. Here are the options and what they do:\n",
+    "\n",
+    "- none: do no transfers and incur no charges on my AWS bill.\n",
+    "- range: Transfer a range of revisions from the ADX to your S3 bucket\n",
+    "- latest: Transfer the latest revision from the ADX to your S3 bucket\n",
+    "- all: Transfer all revisions from the ADX to your S3 bucket\n",
+    "- slowly-download-to-local: Transfer a range of revisions from the ADX to your S3 bucket\n",
+    "\n",
+    "To download a single revision, set the `start_date` and `end_date` to the same day."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d5b35df",
+   "metadata": {},
+   "source": [
+    "------------\n",
+    "\n",
+    "**FPS Critic Inc., owner of PureSkill.gg, is not liable for any AWS\n",
+    "costs your incur. Run the cells below only if you understand and accept\n",
+    "the AWS billing implications.**\n",
+    "\n",
+    "------------"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8290ee4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Optionally set a bucket name. Otherwise, a bucket named [your-account-alias]-pureskillgg-dataset will be created.\n",
```

Comment:
Should be like

```python
# Optionally set a custom bucket name. Bucket names are universally unique.
# Leave this set to None for the default name of [your-account-alias]-pureskillgg-dataset.
# If you want to use an existing bucket, set its name here.
bucket = None

# If you would like this notebook to create the S3 bucket to receive data from the ADX.
# If the bucket already exists, this will have no effect.
create_destination_bucket = True

# Optionally set a prefix for all object keys exported to the bucket, e.g., some/path/.
# We strongly recommend NOT setting this option unless you know what you are doing.
prefix = None

```

Reply:
Added this and fixed up the other two options a bit